TypeCobol/Compiler/Nodes/Attributes.cs

```diff
@@ -267,34 +302,55 @@ public object GetValue(object o, SymbolTable table)
             throw new System.ArgumentOutOfRangeException("Too many receiving items (" + variablesWritten.Count + ")");
         }
     }
-    internal class PointerReceiversAttribute : Attribute
+    internal class PointerDisplayableReceiversAttribute : Attribute
     {
         public object GetValue(object o, SymbolTable table)
         {
             var node = (Node)o;
             var codeElement = node.CodeElement;
             var setStatement = codeElement as SetStatementForIndexes;
-            List<List<DataDefinition>> variablesWrittenRaw = new List<List<DataDefinition>>();
-            List<DataDefinition> variablesWritten = new List<DataDefinition>();
+            List<string> displayableWritten = new List<string>();
             if (setStatement != null && node.IsFlagSet(Node.Flag.NodeContainsPointer))
             {
                 foreach (var data in setStatement.StorageAreaWrites)
```

Comment:
use the new StorageAreaWritesDataDefinition, you will get the DataDefinition directly.